dali/base/dasess.cpp

```diff
@@ -942,6 +942,26 @@ class CClientSessionManager: public CSessionManagerBase, implements IConnectionM
         }
         if (!udesc)
             return SecAccess_Unknown;
+
+        //Verify user signature if present
+        if (udesc->querySignature())
+        {
+            IDigitalSignatureManager * pDSM = createDigitalSignatureManagerInstanceFromEnv();
+            if (pDSM && pDSM->isDigiVerifierConfigured())
+		    {
+		        StringBuffer username;
+		        udesc->getUserName(username);
+		        StringBuffer b64Signature(udesc->querySignature());
+			    if (!pDSM->digiVerify(username.str(), b64Signature))
```

Comment:
indent.

Reply:
I missed this before.  What is the context of this signature that only contains the username?  Only using the username isn't really signing anything as it doesn't include object or document information to make a unique signature.  Singing just user could easily be reused.

Reply:
I opened https://track.hpccsystems.com/browse/HPCC-20041 to address this

Reply:
Ok, still not clear on exactly what this is being used for, but that's a discussion for the new issue.